rbac/internal/integration/views.py

```diff
@@ -50,7 +50,7 @@ def modified_only_filter(self, queryset, field, modified_only):
 class TenantViewSet(viewsets.GenericViewSet, mixins.ListModelMixin):
     """Tenant view set."""
 
-    queryset = Tenant.objects.all()
+    queryset = Tenant.objects.exclude(ready=False, tenant_name="public")
```

Comment:
maybe leave off public tenant exclusion? probably makes the query (much?) less efficient

Reply:
We found there is actually an index on tenant_name after all, so this should be fine